cmd/lncli/walletrpc_active.go

```diff
@@ -745,6 +760,21 @@ func fundPsbt(ctx *cli.Context) error {
 		}
 	}
 
+	if ctx.IsSet("change_type") {
+		switch addressType := ctx.String("change_type"); addressType {
+		case "p2tr":
+			req.ChangeType =
```

Comment:
Since this will need a `///nolint:lll` comment above anyway, we might as well put the value on the same line as the assignment.